requirements.txt

```diff
@@ -0,0 +1,5 @@
+Image==1.5.5
```

Comment:
I don't think we need this requirements.txt file.  I removed it previously because we are now depending on the install_conda_deps.sh